pages/agenda/2022.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react'
+import { NextPage, GetStaticProps } from 'next'
+import { From2022 } from 'config/2022'
+import { StyledVideoContainer } from 'components/Agenda/Agenda.styled'
+import { AgendaForDisplay, ServiceSlot, SessionSlot, Session, SponsorType } from 'config/types'
+import { Sponsors } from 'components/Sponsors/sponsors'
+import ResponsiveVideo from 'components/responsiveVideo'
+import { SafeLink } from 'components/global/safeLink'
+import { Text } from 'components/global/text'
+import { ButtonAnchor } from 'components/global/Button/Button'
+import { zonedTimeToUtc } from 'date-fns-tz'
+import { useConfig } from 'Context/Config'
+import { Main } from 'layouts/main'
+import AllAgendas from 'components/allAgendas'
+
+import data from 'public/static/agenda/2022.json'
+import sessionizeGridSmart from 'public/static/agenda/2022_gridsmart.json'
```

Comment:
Considering changing this so that the static file is the `AgendaForDisplay` type, but I'll do that after the API returns updated date

Reply:
Doing so means finding a better spot for the `with2022Overrides` function